app/src/main/java/com/yagni/view/ManageInvGUI.java

```diff
@@ -9,365 +9,447 @@
+
+            // Calls the readAll method and passes in the db connection
+            read.readAll(linkDB);
 
+        } else if (e.getSource() == addButton) {
+            // add button set variables to user input from text fields
+            final String productId = productIDField.getText();
+            final String quantity = quantityField.getText();
+            final String wholeSale = wholeSaleField.getText();
+            final String salePrice = salePriceField.getText();
+            final String supplierId = supplierIDField.getText();
+
+            // Clear all text fields
             clearTextFields();
 
-            // creating a Creat obj and calling addRecord passing in user input
+            // Creating the object to call addRecord and passing in user input
             Create addRecord = new Create(productId, quantity, wholeSale, salePrice, supplierId);
             addRecord.addRecord(linkDB);
+
+            // Reload product inventory data table to display newly added data
             refreshProducts();
 
         } else if (e.getSource() == clearButton) {
+            // On click of the clear button clear all text fields
             clearTextFields();
 
         } else if (e.getSource() == updateButton) {
-			// setting our variables to user input
-            String updateID = productIDField.getText();
-            String updateQuant = quantityField.getText();
-            String updateWholesale = wholeSaleField.getText();
-            String updatePrice = salePriceField.getText();
-            String updateSupplierID = supplierIDField.getText();
-
-		    Update updateRecord = new Update(updateID, updateQuant, updateWholesale, updatePrice, updateSupplierID);
+            // On click of the update button set variables to user input from text fields
+            final String updateID = productIDField.getText();
+            final String updateQuant = quantityField.getText();
+            final String updateWholesale = wholeSaleField.getText();
+            final String updatePrice = salePriceField.getText();
+            final String updateSupplierID = supplierIDField.getText();
+
+            // Creating the object to call update and passing in user input
+            Update updateRecord = new Update(updateID, updateQuant, updateWholesale, updatePrice, updateSupplierID);
             updateRecord.update(linkDB);
+
+            // Reload product inventory data table to display updated data
             refreshProducts();
 
-		} else if (e.getSource() == reloadButton) {
+        } else if (e.getSource() == reloadButton) {
+            // On click of the reload button refresh the products table
             refreshProducts();
 
         } else if (e.getSource() == logoutButton) {
-            tablePanel.remove(reloadButton);
-            tablePanel.repaint();
-            tablePanel.add(loadButton);
+            // On click of the logout button close the db connection and dispose the
+            // JFrame
```

Comment:
move this //JFrame comment up to join in to 305. Line-wrap shouldn't be needed.

Reply:
So this auto formats to wrap the comment because of the 4.4/4.5 Column limit/line wrapping. I have changed this comment to be less than 100 characters so it is all on one line to avoid wrapping by auto-formatting.